workos/resources/directory_sync.py

```diff
@@ -92,3 +97,12 @@ def to_dict(self):
         directory_group = super(WorkOSDirectoryUser, self).to_dict()
 
         return directory_group
+
+    def primary_email(self, user):
```

Comment:
Since this is an instance method we shouldn't take in the `user` here, instead we should return the primary email of the user held in `self`.